docs/edgeql/functions/converters.rst

```diff
@@ -14,7 +14,7 @@ the functions below allow more options for such conversions.
 
     Return JSON value represented by the input *string*.
 
-    This is the reverse of :eql:func:`json_to_str`.
+    This is the reverse of :eql:func:`to_str`.
```

Comment:
Maybe "This is the reverse" ->  "see also"?

Reply:
Sure